shared/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-multiplatform'
 apply plugin: 'kotlin-kapt'
+apply plugin: 'realm-android'
```

Comment:
I don't understand gradle enough to know wether this file is optimal or not. I'm sure it could be cleaned up and I'll see what I can do, but it does otherwise work.